langs/lambda_node.go

```diff
@@ -4,6 +4,22 @@ type LambdaNodeHelper struct {
 	BaseHelper
 }
 
+func (h *LambdaNodeHelper) Handles(lang string) bool {
+	return defaultHandles(h, lang)
+}
+func (h *LambdaNodeHelper) Runtime() string {
+	return h.LangStrings()[0]
+}
+
+func (lh *LambdaNodeHelper) LangStrings() []string {
+	return []string{"lambda-nodejs4.3", "lambda-node-4"}
```

Comment:
isn't this a good time to add version tag to lambda-node as well?

Reply:
I think lambda helper will go away soon, instead support it in FDK. But regardless, I didn't do anything with versioning here.